dev-lang/perl/perl-5.34.0-r5.ebuild

```diff
@@ -518,7 +518,7 @@ src_configure() {
 	[[ ${COLUMNS:-1} -ge 1 ]] || unset COLUMNS # bug #394091
 
 	# Perl has problems compiling with -Os in your flags with glibc
-	use elibc_uclibc || replace-flags "-Os" "-O2"
+	replace-flags "-Os" "-O2"
```

Comment:
I don't expect you to necessarily try this but if it's "with glibc", it really should've been ``use elibc_glibc && ...`` in the first place..

@akhuettel, any ideas?